src/m365/aad/commands/app/app-add.ts

```diff
@@ -50,6 +51,7 @@ interface CommandArgs {
 interface Options extends GlobalOptions {
   apisApplication?: string;
   apisDelegated?: string;
+  grantAdminConsent?: boolean;
```

Comment:
Could you also add the property to the telemetryOptions?

Reply:
Thanks, always forget about this one...